tests/unit/dom/wait-for-test.js

```diff
@@ -71,11 +71,11 @@ module('DOM Helper: waitFor', function(hooks) {
 
     let start = Date.now();
     try {
-      await waitFor('.something', { timeout: 100 });
+      await waitFor('.something', { timeout: 100, timeoutMessage: '.something timed out' });
```

Comment:
Lets leave this test alone, and add another that is specifically for modifying the timeout message...